src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -81,6 +76,11 @@ public UniquePersonList(UniquePersonList source) {
         return Collections.unmodifiableList(internalList);
     }
 
+    public List<ReadOnlyPerson> sortedListView() {
```

Comment:
Missing header comment. The name of this method a little ambiguous - so does it actually sort the list or just return a sorted view of the list, without touching the original data? Seems like the former here.